File: frontend/prest/src/app/models/newsRestaurant/NewsRestaurantUpdatePayload.ts
import {Payload} from "../common/Payload";
import {Restaurant} from "@restaurant/Restaurant";
import {User} from "@user/User";

export interface NewsRestaurantUpdatePayload extends Payload
{
  id_news_restaurant: number;
  text: string;
  restaurant: Restaurant;
  user: User;
  date: Date;
}

File: Prest/src/main/java/be/prest/Payload/Update/NewsRestaurantUpdatePayload.java
package be.prest.Payload.Update;

import be.prest.Entities.*;
import lombok.*;

import java.sql.*;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class NewsRestaurantUpdatePayload {
    private int id_news_restaurant;
    private String text;
    private Timestamp date;
    private Boolean actif;
    private Restaurant restaurant;
    private User user;

}
